internal/pkg/aws/codepipeline/codepipeline.go

```diff
@@ -276,10 +277,12 @@ func (ss *StageState) HumanString() string {
 	if transition == "" {
 		transition = empty
 	}
+	tree := treeprint.New()
```

Comment:
Can you move this declaration under the stageString declaration?

Reply:
yup, done!